Cargo.toml

```diff
@@ -32,7 +32,7 @@ fuzztarget = []
 
 [dev-dependencies]
 rand = "0.6"
-rand_core = "0.3"
```

Comment:
Is there a specific reason why you bump version? Your PR compiles with 0.3 too.

Reply:
I think it's fine because it's only a dev-dependency.